src/patterns/SaleHeaderBanner/_saleHeaderBanner.scss

```diff
@@ -23,7 +23,8 @@
     padding: $spacing-md 0;
 
     &__mobile-countdown {
-      display: block;
+      display: flex;
+      width: 100%;
```

Comment:
as an alternative to `width: 100%`, you can change the parent `.seldon-sale-header-banner__stack` from `align-items: center` to `align-items: stretch;` and then set the button to `align-self: flex-start`